MessengerClone/FetchImage.swift

```diff
@@ -0,0 +1,149 @@
+import SwiftUI
+import Nuke
+
+/// Taken from: https://github.com/kean/FetchImage
```

Comment:
We should also include the original copyright

Reply:
I added the header comments from the original file above this comment. I was not able to change the top-level header because that is rewritten by `swiftformat`. I could probably find a way how to exclude but I believe the current situation might be fine

Reply:
Yep, works 👍